src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,59 @@
+import React, {memo} from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import styles from '../styles/styles';
+import Text from './Text';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+
+const propTypes = {
+
+    isDefaultChatRoom: PropTypes.bool,
+    chatUsers: PropTypes.arrayOf(PropTypes.object),
+    ...withLocalizePropTypes,
+
+};
+
+const defaultProps = {
+    chatUsers: [],
+    isDefaultChatRoom: false,
+};
+
+const ChatBeginingText = ({isDefaultChatRoom, chatUsers, translate}) => (
+    <Text style={[styles.mt3, styles.w70, styles.textAlignCenter]}>
+        <Text>
+            {isDefaultChatRoom ? `${translate('reportActionsView.beginingOfChatHistroyPrivate')} ` : `${translate('reportActionsView.beginingOfChatHistroy')} `}
+        </Text>
+        {isDefaultChatRoom
+            && (
+                <Text>
+                    {`${lodashGet(chatUsers, 'chatUsers[0].displayName', '')} ${translate('reportActionsView.beginingOfChatHistroyPrivateSectionPart')}`}
+                </Text>
+            )}
+        {!isDefaultChatRoom
+            && (
+                <Text>
+                    {chatUsers.map(({displayName, pronouns}, index) => (
+                        <Text key={displayName}>
+                            <Text style={[styles.chatTextStyle]}>
+                                {displayName}
+                            </Text>
+                            <Text>
+                                {` (${pronouns})`}
```

Comment:
if `pronouns` is empty then you can simply use boolean check to hide this elment. Either `Boolean(pronouns)` or `!!pronouns`.